packages/main/src/plugin/proxy-resolver.spec.ts

```
@@ -22,30 +22,29 @@
 import { get } from 'node:http';
 import * as nodeurl from 'node:url';
 
-import type { HttpsProxyAgentOptions } from 'hpagent';
+import type * as hpagent from 'hpagent';
 import { HttpsProxyAgent } from 'hpagent';
 import { beforeEach, expect, test, vi } from 'vitest';
 
 import type { Certificates } from './certificates.js';
 import type { Proxy } from './proxy.js';
```

Biome (2.4.11)

[error] Do not shadow the global "Proxy" property.

(lint/suspicious/noShadowRestrictedNames)